Simplenote.xcodeproj/project.pbxproj

```diff
@@ -4569,6 +4577,7 @@
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_ENTITLEMENTS = SimplenoteIntents/SimplenoteIntentsDebug.entitlements;
```

Comment:
These changes were done via the Xcode UI:

![image](https://user-images.githubusercontent.com/1218433/128134886-ebc82a58-5357-4c0d-905d-26318364b5be.png)